doc/reference_doc_guidelines.md

```diff
@@ -0,0 +1,299 @@
+# Azure CLI reference and conceptual content guidelines
+
+## General guidelines
+
+General guidelines apply to both reference and conceptual content.
+
+- Develop and test all code blocks in both a Bash and PowerShell environment. When commands have syntax differences between environments, create one example for Bash and another for PowerShell. For copyability, use different code blocks with a notation. Here is an example:
+
+    ```azurecli-interactive
+    az command –-p1 written-for-Bash 
+    ```
+
+    ```azurecli-interactive
+    # This same command can be run in PowerShell, but with a slight syntax difference 
+    az command –-p1 written-for-PowerShell-showing-quoting-differences 
+    ```
+
+  For more information on syntax differences between Bash and PowerShell, see these articles:
+  - [Quoting issues with PowerShell](https://github.com/Azure/azure-cli/blob/dev/doc/quoting-issues-with-powershell.md)
+  - [Use quotation marks in Azure CLI parameters](./use-cli-effectively.md#use-quotation-marks-in-parameters)
+  - Compare syntax of CMD, PowerShell and Bash in [Query command output using JMESPath](./query-azure-cli.md)
+  - [Error handling for the Azure CLI in PowerShell](./use-cli-effectively.md#error-handling-for-azure-cli-in-powershell)
+
+    **Do not replicate code blocks for line break differences.** The new Azure CLI code block <kbd>Copy</kbd> button removes Bash and PowerShell line break characters. Only replicate code blocks for syntax differences.
+
+## Conceptual content
+
+### Minimum content
+
+Minimum conceptual content for the Azure CLI is one conceptual article for every command group and subgroup that creates or adds a billable Azure resource. For example:
```

Comment:
Which team will add/update conceptual articles? I assume your team will help on that, is it true?